javascript/js-fetch/js-fetch_color-clue/js/fetch.js

```diff
@@ -18,6 +18,12 @@ export async function fetchNewColor() {
    */
 
   // --v-- your code here --v--
-
+  try {
+    const response = await fetch(colorApiUrl);
+    const data = await response.json();
+    setColorToGuess(data.hex.value, data.name.value);
```

Comment:
As far as I understood the instruction instead of data.hex.value it should be `.name.closest_named_hex`. 
See here:
   * The function takes two arguments:
   *  - the hex code of the closest named color
   *  - and the name of the random color

Reply:
Ок, thanks. I alredy fix it.